### ISSUE TITLE
Migrate crash when bumping from 2.12.0  [node15]

### ISSUE THREAD

**Issue by himynameisdave** [prisma/prisma#5137]
Hi Prisma Team! Prisma Migrate just crashed. 

I am trying to follow these [docs](https://www.prisma.io/docs/guides/prisma-guides/prisma-migrate-guides/add-prisma-migrate-to-a-project) on upgrading from the `experimental`, from version `2.12.1`.

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v15.6.0            | 
| Prisma CLI  | 2.14.0             | 
| Binary      | 5d491261d382a2a5ffdc71de17072b0e409f1cc1| 

## Error
```
Error: [libs/datamodel/connectors/dml/src/datamodel.rs:172:14] Every RelationInfo should have a complementary RelationInfo on the opposite relation field.
```

---

Here's my schema:

```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}

model Artist {
  id          String    @default(uuid()) @id
  name        String    @unique
  albums      Album[]   @relation(references: [id])
  reviews     Review[]  @relation(references: [id])
  createdAt   DateTime  @default(now())
  updatedAt   DateTime  @updatedAt
}

model Album {
  id          String    @default(uuid()) @id
  title       String
  albumArtURL String
  year        Int
  artists     Artist[]  @relation(references: [id])
  genres      Genre[]   @relation(references: [id])
  labels      Label[]   @relation(references: [id])
  score       Float
  review      Review    @relation(fields: [reviewId], references: [id])
  reviewId    String
  createdAt   DateTime  @default(now())
  updatedAt   DateTime  @updatedAt
}

model Genre {
  id          String    @default(uuid()) @id
  name        String    @unique
  albums      Album[]   @relation(references: [id])
}

model Label {
  id          String    @default(uuid()) @id
  name        String    @unique
  albums      Album[]   @relation(references: [id])
}

model Review {
  id               String      @default(uuid()) @id
  url              String      @unique
  date             DateTime
  artists          Artist[]    @relation(references: [id])
  albums           Album[]
  text             ReviewText  @relation(fields: [textId], references: [id])
  textId           String
  authors          Author[]    @relation(references: [id])
  createdAt        DateTime    @default(now())
  updatedAt        DateTime    @updatedAt
}

model ReviewText {
  id               String      @default(uuid()) @id
  abstract         String
  text             String
}

model Author {
  id               String      @default(uuid()) @id
  name             String      @unique
  title            String
  url              String?
  reviews          Review[]    @relation(references: [id])
  createdAt        DateTime    @default(now())
  updatedAt        DateTime    @updatedAt
}
```

**Comment by pantharshit00** [comment]
Hey @himynameisdave 


I am unable to reproduce the error with the schema. Can you please share the migration files in git repository so that we can inspect those?

**Comment by himynameisdave** [comment]
Hey all! I've recently fixed this issue (I bumped to `2.16.x` and just re-ran a migration following [this guide](https://www.prisma.io/docs/guides/prisma-guides/add-prisma-migrate-to-a-project)).

Not sure how I was hitting this issue before (I think with `prisma introspect` maybe?). In any case, thanks for looking into this!
